sqs/sqs_test.go

```diff
@@ -0,0 +1,47 @@
+package sqs
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/stretchr/testify/assert"
+)
+
+type MockSQS struct {
+	QueueAttributes *sqs.GetQueueAttributesOutput
+}
+
+func (m *MockSQS) GetQueueAttributes(*sqs.GetQueueAttributesInput) (*sqs.GetQueueAttributesOutput, error) {
+	return m.QueueAttributes, nil
+}
+
+func (m *MockSQS) SetQueueAttributes(input *sqs.SetQueueAttributesInput) (*sqs.SetQueueAttributesOutput, error) {
```

Comment:
similar comments to those i made before - you can accept the attributes directly, and you don't need to return an error: this method only exists to make the mock controllable, not to satisfy an interface.